python/pyspark/sql/connect/client/core.py

```diff
@@ -450,18 +486,18 @@ def keys(self) -> List[str]:
 
 class AnalyzeResult:
     def __init__(
-        self,
-        schema: Optional[DataType],
-        explain_string: Optional[str],
-        tree_string: Optional[str],
-        is_local: Optional[bool],
-        is_streaming: Optional[bool],
-        input_files: Optional[List[str]],
-        spark_version: Optional[str],
-        parsed: Optional[DataType],
-        is_same_semantics: Optional[bool],
-        semantic_hash: Optional[int],
-        storage_level: Optional[StorageLevel],
+            self,
```

Comment:
Hm, we won;t need formatting here. Was it by python auto formatter?

Reply:
Used autoformatter and it get back to reason, so probably IDE did. I'm changing it back, sorry :)